addthis_block/addthis_block.info.yml

```diff
@@ -0,0 +1,11 @@
+name: AddThis Block
+description: Provides a configurable block to render AddThis elements.
```

Comment:
Is there a reason why this is a submodule? Just curious what the motivation is and if this description can better reflect that